gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -374,11 +374,11 @@
         </module>
         <module name="RightCurly"> <!-- Java Style Guide: Nonempty blocks: K & R style -->
             <property name="option" value="same"/>
-            <property name="tokens" value="LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE, LITERAL_DO"/>
+            <property name="tokens" value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE, LITERAL_DO"/>
```

Comment:
this is very weird - this didn't use to complain with checkstyle 8.13 on codebases I tried but with 8.33 this fails. I would be tempted to just ditch this since palantir-java-format is the autoritative source. I have made this setting conform to what palantir-java-format generates.